### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ermolenkodev**
* Update QuickStart doc with android info

* Add ONNX inference example doc

* Update Preprocessing DSL in transfer learnig docs

* Add doc with all available ONNX models

### CODE REVIEW

File: docs/onnx_model_zoo.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## KotlinDL ONNX Model Zoo
+
+| Model                     |     Android support      |
```
Comment (juliabeliaeva) [inline docs/onnx_model_zoo.md:3]: Intellij suggests to reformat this table:
image

File: docs/onnx_model_zoo.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+|---------------------------|:------------------------:|
+| EfficientNet4Lite         |    :heavy_check_mark:    |
```
Comment (juliabeliaeva) [inline docs/onnx_model_zoo.md:5]: Perhaps, we could group models by task (classification, object detection, pose detection, etc)?

**Review by juliabeliaeva — APPROVED** [review]

File: docs/inference_onnx_model.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+### Inference of the model included into KotlinDL Model Zoo
+In this section, the same pose detection model will be used.
+One more thing to mention is that KotlinDL expects the model files to be located in the application resources. You can download the required models using Gradle plugin or manually. *TODO update*
+
+
+Also note that input data type is `Bitmap` instead of `BufferedImage`.
+```kotlin
+val modelHub = ONNXModelHub(context) // Android context is required to access the application resources
+val model = ONNXModels.PoseDetection.MoveNetMultiPoseLighting.pretrainedModel(modelHub)
```
Comment (juliabeliaeva) [inline docs/inference_onnx_model.md:78]: Just noticed that the tutorial describes `MoveNetMultiPoseLighting` which is not available on android. Do we want to convert it for android or should we change the tutorial?

Comment (ermolenkodev) [reply docs/inference_onnx_model.md:78]: Yes, it's a mistake. Thank you for noticing it!

**Pull request merged**